doozer/doozerlib/cli/scan_fips.py

```diff
@@ -0,0 +1,55 @@
+"""
+For this command to work, https://github.com/openshift/check-payload binary has to exist in PATH and run as root
+"""
+import json
+
+import click
+import koji
+from doozerlib.cli import cli, pass_runtime
+from doozerlib.runtime import Runtime
+from typing import Optional
+from doozerlib.exectools import cmd_gather
+
+
+class ScanFipsCli:
+    def __init__(self, runtime: Runtime, nvrs: Optional[list]):
+        self.runtime = runtime
+        self.nvrs = nvrs
+
+        # Initialize runtime and brewhub session
+        self.runtime.initialize(clone_distgits=False)
+        self.koji_session = koji.ClientSession(self.runtime.group_config.urls.brewhub)
+
+    def run(self):
+        # Get the list of NVRs to scan for
+        # (nvr, pull-spec) list of tuples
+        image_pullspec_mapping = []
+
+        for nvr in self.nvrs:
```

Comment:
We could use `exectools.parallel_exec()` to get some parallelism and save some time. If you think this is worth it, you can see an example [here](https://github.com/openshift-eng/art-tools/blob/main/doozer/doozerlib/cli/scan_sources.py#L167)

Reply:
getBuild is usually pretty quick. So I think we can skip for now. Once the feature is out, I'll test this out against a mass rebuild and see how it goess